assets/js/components/ErrorText.js

```diff
@@ -67,17 +68,20 @@ function ErrorText( { message, reconnectURL } ) {
 			<p
 				dangerouslySetInnerHTML={ sanitizeHTML( error, sanitizeArgs ) }
 			/>
+			{ children && <Fragment>&nbsp;{ children }</Fragment> }
```

Comment:
@sashadoes I'am not a big fan of `&nbsp;` for spacing.
```suggestion
			{ children && <Fragment> { children }</Fragment> }
```
works for me. Notice the whitespace between `<Fragment>` and `{ children }`